rpm/droid-hal-kirin.spec

```diff
@@ -3,4 +3,7 @@
 %define lunch_device aosp_i4113-user
 
 %include rpm/droid-hal-common.inc
+
+Source0: %{device}-%{version}.tar.bz2
```

Comment:
Should be `Source0: droid-hal-%{device}-%{version}.tar.bz2`

Reply:
ah, my bad, will fix

Reply:
And it all can go to -common.inc (as it tested fine to resolve), just below droid-src-... macro

Reply:
But does it resolve properly on promotion? Are you sure? :)

Reply:
Nothing is sure on promotions :D
The original problem is that promotions fail if macro is defined inside a conditional.
If this line is within -common.inc, it's just simple cascade of includes, where %{device} will already be defined.

Reply:
Ah, true, so it might work. Let's try simplified version first. Will fix.